### PR TITLE
Remove the usage of default value to fix invalid proto3 files.

### PR DESCRIPTION
**Pull request by linkerzhang**
This PR contains some format changes, as it's a very easy change so that I keep all format changes (suppose we should always run clang-format on each file before checking in so that format is always good).

Adding CI to cover this check will be done later. I'm figuring out how to do that, our linux ci protoc can only recognize proto2 and protoc can't be recognized in windows ci env :(.

### CODE REVIEW

File: onnx/common/constants.h [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+#include <string>
+
+namespace ONNX_NAMESPACE {
+namespace Common {
+// For ONNX op/function registration.
+
+// ONNX domains.
+constexpr const char* AI_ONNX_ML_DOMAIN = "ai.onnx.ml";
+constexpr const char* ONNX_DOMAIN = "";
```
Comment (bddppq) [inline onnx/common/constants.h]: Hmm, such move is a breaking change

File: onnx/defs/math/old.cc [hunk 2]
```diff
@@ -5,7 +5,7 @@
 #include "onnx/defs/schema.h"
 
 namespace ONNX_NAMESPACE {
-
+using namespace Common;
```
Comment (houseroad) [inline onnx/defs/math/old.cc]: I think `using Common::ONNX_DOMAIN;` should be enough in most of the cases.

File: onnx/common/constants.h (hunk 1 above)
Comment (linkerzhang) [reply onnx/common/constants.h]: :). OK, removed the namespace Common.

File: onnx/defs/math/old.cc (hunk 2 above)
Comment (linkerzhang) [reply onnx/defs/math/old.cc]: Common namespace is not used for now.

**Comment by linkerzhang** [comment]
Comments resolved. Merging it.

**Pull request merged**